add.go

```diff
@@ -110,7 +114,7 @@ func getURL(src string, chown *idtools.IDPair, mountpoint, renameTarget string,
 	}
 	// Figure out the size of the content.
 	size := response.ContentLength
-	responseBody := response.Body
+	var responseBody io.Reader = response.Body
```

Comment:
Not-blocker: (Asking for info) Is there any special reason for this line change ? I mean why do need to explicitly set the type to `io.Reader` ?

Reply:
nvm I think TeeReader is using this.

Reply:
> nvm I think TeeReader is using this.

Yep that's why.